src/core/BUILD

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bazel:grpc_build_system.bzl",
+    "grpc_cc_library",
+    "grpc_generate_one_off_targets",
+    "grpc_upb_proto_library",
+    "grpc_upb_proto_reflection_library",
+    "python_config_settings",
+)
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+exports_files(glob(["**"]))
+
+licenses(["reciprocal"])
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "layering_check",
+        "-parse_headers",
+    ],
+)
+
+grpc_cc_library(
```

Comment:
Can we do `event_engine_common` before getting too much further... it's got a dependency on `include/` that I think is going to inform the rest of the decisions we might make on this.

Reply:
Thanks for the suggestion!